plugins/org.jkiss.dbeaver.data.transfer.ui/src/org/jkiss/dbeaver/tools/transfer/ui/pages/stream/StreamConsumerPageOutput.java

```diff
@@ -276,7 +276,7 @@ public void activatePage() {
 
         clipboardCheck.setSelection(settings.isOutputClipboard());
         singleFileCheck.setSelection(settings.isUseSingleFile());
-        appendToEndOfFileCheck.setSelection(settings.isAppendToFileEnd());
+        appendToEndOfFileCheck.setSelection(!isBinary && settings.isAppendToFileEnd());
```

Comment:
Other conditions are not met, please take look at:
https://github.com/dbeaver/dbeaver/blob/9667baa4d6070e25090a9513136c532a2010fef8/plugins/org.jkiss.dbeaver.data.transfer.ui/src/org/jkiss/dbeaver/tools/transfer/ui/pages/stream/StreamConsumerPageOutput.java#L259